substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java

```diff
@@ -81,7 +81,12 @@ public boolean consume(ArgumentQueue args) {
             case "--version":
                 args.poll();
                 singleArgumentCheck(args, headArg);
-                String message = System.getProperty("java.vm.version");
+                String message;
+                if (NativeImage.IS_AOT) {
+                    message = System.getProperty("java.vm.version");
+                } else {
+                    message = "native-image " + NativeImage.graalvmVersion + " " + System.getProperty("org.graalvm.config", "CE");
```

Comment:
You could use `NativeImage.graalvmConfig` here.
```suggestion
                    message = "native-image " + NativeImage.graalvmVersion + " " + NativeImage.graalvmConfig;
```

Reply:
Fixed, thanks!